### PR TITLE
WIP Progress stash

### PR DESCRIPTION
**Pull request by Fohlen**
Add's nodegit progress and command line progress bars to flex.

Please squash/remove the "merge to master" thing if possible @a-teammate

### CODE REVIEW

File: src/releases/ReleaseManager.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ class ReleaseManager extends EventEmitter {
         } else if (os.platform() == 'linux') {
             platform += 'Linux';
         } else if (os.platform() == 'darwin') {
-            platform += 'Darwin';
+            platform += 'Windows';
```
Comment (a-teammate) [inline src/releases/ReleaseManager.js]: Ehm :D

**Comment by a-teammate** [comment]
could we merge this please? :)

the littel fix Windows -> Darwin is required, otherwise merge and publish?

**Comment by Fohlen** [comment]
No, the download progress is actually broken/wrong for release downloads. Didn't dig in it for some days, needs to be fixed before we're ready

**Comment by Fohlen** [comment]
Can we go for this?

**Pull request merged**